cmd/kubeadm/app/preflight/checks.go

```diff
@@ -1092,15 +1092,41 @@ func TryStartKubelet(ignorePreflightErrors sets.String) {
 	// If we notice that the kubelet service is inactive, try to start it
 	initSystem, err := initsystem.GetInitSystem()
 	if err != nil {
-		glog.Infoln("[preflight] no supported init system detected, won't ensure kubelet is running.")
-	} else if initSystem.ServiceExists("kubelet") {
-
-		glog.Infoln("[preflight] Activating the kubelet service")
-		// This runs "systemctl daemon-reload && systemctl restart kubelet"
-		if err := initSystem.ServiceRestart("kubelet"); err != nil {
-			glog.Warningf("[preflight] unable to start the kubelet service: [%v]\n", err)
-			glog.Warningf("[preflight] please ensure kubelet is running manually.")
-		}
+		glog.Infoln("[preflight] no supported init system detected, won't make sure the kubelet is running properly.")
+		return
+	}
+
+	if !initSystem.ServiceExists("kubelet") {
+		glog.Infoln("[preflight] couldn't detect a kubelet service, can't make sure the kubelet is running properly.")
+	}
+
+	glog.Infoln("[preflight] Activating the kubelet service")
+	// This runs "systemctl daemon-reload && systemctl restart kubelet"
+	if err := initSystem.ServiceRestart("kubelet"); err != nil {
+		glog.Warningf("[preflight] unable to start the kubelet service: [%v]\n", err)
```

Comment:
This seems like an error instead of a warning, unless there's a way we can recover automatically from this failure?

Additionally, instead of using glog.Warningf, please add a warning prefix and use `fmt.Println`. Or if you feel like warning isn't the right level, use `fmt.Printf` or `glog.V(x).Infof`